identity/aziot-identityd/src/http/create_or_list_module_identity.rs

```diff
@@ -15,14 +16,23 @@ impl http_common::server::Route for Route {
     fn from_uri(
         service: &Self::Service,
         path: &str,
-        _query: &[(std::borrow::Cow<'_, str>, std::borrow::Cow<'_, str>)],
+        query: &[(std::borrow::Cow<'_, str>, std::borrow::Cow<'_, str>)],
     ) -> Option<Self> {
         if path != "/identities/modules" {
             return None;
         }
 
+        let id_type: Option<String> = query.iter().find_map(|q| {
+            if q.0.to_lowercase() == "type" {
```

Comment:
We don't treat api-version case-insensitively, so we shouldn't treat this case-insensitively either.

Reply:
Fixed